cmd/project.go

```diff
@@ -10,13 +10,16 @@ import (
 
 var projectBuildCmdSrc string
 var projectBuildCmdDesc string
+var projectBuildCmdExtensions []string
```

Comment:
why do we need extensions here if extensions can't influence of project build functionality?

Reply:
Like in the provide modules case, I think it's better to send the list of extensions to all commands that parse the mta.yaml instead of going through the logic of what part of the MTA object each one uses and what the extensions can change